portfolio/src/main/java/com/google/sps/servlets/FactsServlet.java

```diff
@@ -0,0 +1,48 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.gson.Gson;
+import com.google.sps.data.FactsData;
+
+/** Handles requests sent to the /facts URL. Try running a server and navigating to /hello! */
+@WebServlet("/facts")
+public class FactsServlet extends HttpServlet {
+
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    //generate the array list
+    final ArrayList<FactsData> Facts = new ArrayList<FactsData>();
```

Comment:
Nit: variable names in Java typically use camel case (i.e. a lowercase first letter, then capitalize the first letter of any words later in the name). So this ArrayList would be named `facts` instead of `Facts`! 

If it had multiple words, camel case would look like this: `myLongNameExample`

Reply:
I see! I like using camel cases as well! I think this `Facts` was a mistake:)